zanata-client-commands/src/main/resources/prompts.properties

```diff
@@ -89,3 +89,5 @@ no.default.mapping=Can not find default translation file mapping rule for projec
 invalid.rule=Rule defined is not valid (must contain locale): %s
 unrecognized.variables=Valid variables are %s. Anything else in '%s' will be treated as literal value.
 confirm.rule=Continue with current mapping rule(s) (y/n)?
+
+locales.in.config.deprecated=Locale mappings in project config file (zanata.xml) is deprecated! Please remove it from the file so that it can use latest language settins on the server.
```

Comment:
I will update the message. The key information it should include is:

 - locale mappings are now handled on the server
 - you should add a locale alias on the server for each locale mapping in zanata.xml
 - after adding aliases, you should remove <locales> from zanata.xml so that server locales will be used.